companion/index.js

```diff
@@ -31,7 +31,7 @@ if (me.launchReasons.settingsChanged) {
 function sendValue(key, val) {
     if (val) {
         // check that the change was a change in the flow
-        if (key === "flow_index" || key === "buzz_time") {
```

Comment:
May need to revert this if we bring back the flow index

Reply:
Let's sort out whether we want the user to be able to select the questions and then we can decide whether we want to bring it back.